geonetnz_quakes flaky tests

<!-- READ THIS FIRST:
- If you need additional help with this template please refer to https://www.home-assistant.io/help/reporting_issues/
- Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/home-assistant/releases
- Frontend issues should be submitted to the home-assistant-polymer repository: https://github.com/home-assistant/home-assistant-polymer/issues
- iOS issues should be submitted to the home-assistant-iOS repository: https://github.com/home-assistant/home-assistant-iOS/issues
- Do not report issues for integrations if you are using a custom integration: files in <config-dir>/custom_components
- This is for bugs only. Feature and enhancement requests should go in our community forum: https://community.home-assistant.io/c/feature-requests
- Provide as many details as possible. Paste logs, configuration sample and code into the backticks. Do not delete any text from this template!
-->

**Home Assistant release with the issue:**
<!--
- Frontend -> Developer tools -> Info
- Or use this command: hass --version
-->
0.98.0.dev0

**Last working Home Assistant release (if known):**
0.97

**Operating environment (Hass.io/Docker/Windows/etc.):**
<!--
Please provide details about your environment.
-->

**Component/platform:**
<!--
Please add the link to the documentation at https://www.home-assistant.io/components/ of the component/platform in question.
-->
geonetnz_quakes

**Description of problem:**
There are at least three flaky tests merged to dev.

See:
https://dev.azure.com/home-assistant/Home%20Assistant/_build/results?buildId=6243&view=logs&jobId=641e29f4-92ec-5dde-69c3-b45a7bd26967

**Problem-relevant `configuration.yaml` entries and (fill out even if it seems unimportant):**
```yaml

```

**Traceback (if applicable):**
```

```

**Additional information:**

```
ERROR    homeassistant.components.geo_location:entity_platform.py:188 Error while setting up platform geonetnz_quakes |  
-- | --
  | Traceback (most recent call last): |  
  | File "/__w/1/s/homeassistant/helpers/entity_platform.py", line 149, in _async_setup_platform |  
  | await asyncio.wait_for(asyncio.shield(task), SLOW_SETUP_MAX_WAIT) |  
  | File "/usr/local/lib/python3.6/asyncio/tasks.py", line 358, in wait_for |  
  | return fut.result() |  
  | File "/__w/1/s/homeassistant/components/geonetnz_quakes/geo_location.py", line 65, in async_setup_entry |  
  | entry.data[CONF_MINIMUM_MAGNITUDE], |  
  | File "/__w/1/s/homeassistant/components/geonetnz_quakes/geo_location.py", line 99, in __init__ |  
  | filter_time=DEFAULT_FILTER_TIME_INTERVAL, |  
  | File "/__w/1/s/venv/lib/python3.6/site-packages/aio_geojson_geonetnz_quakes/feed_manager.py", line 23, in __init__ |  
  | filter_time=filter_time) |  
  | TypeError: Can't instantiate abstract class GeonetnzQuakesFeed with abstract methods _extract_from_feed
```

OK. I can already see 1 issue with the way I set up the integration library: The integration depends on a specific version of `aio_geojson_geonetnz_quakes`, but this in turn just depends on a minimum version of a more generic JSON library `aio_geojson_client>=0.4`. The intention of this approach was to be able to reuse that generic library in more integration-specific libraries in the future. And where necessary introduce new features in that generic library without breaking any dependant libraries.

I have continued evolving the generic library and did not think about breaking changes. Unfortunately in this case there is now `aio_geojson_client` v0.6 which is incompatible with `aio_geojson_geonetnz_quakes` v0.6. :(

I'll create a new combination of the 2 libraries now, and try to avoid breaking changes in the future.

--- a/homeassistant/components/geonetnz_quakes/manifest.json
+++ b/homeassistant/components/geonetnz_quakes/manifest.json
@@ -4,7 +4,7 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/components/geonetnz_quakes",
   "requirements": [
-    "aio_geojson_geonetnz_quakes==0.5"
+    "aio_geojson_geonetnz_quakes==0.9"
   ],
   "dependencies": [],
   "codeowners": [

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -115,7 +115,7 @@ adguardhome==0.2.1
 afsapi==0.0.4
 
 # homeassistant.components.geonetnz_quakes
-aio_geojson_geonetnz_quakes==0.5
+aio_geojson_geonetnz_quakes==0.9
 
 # homeassistant.components.ambient_station
 aioambient==0.3.1

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -43,7 +43,7 @@ YesssSMS==0.2.3
 adguardhome==0.2.1
 
 # homeassistant.components.geonetnz_quakes
-aio_geojson_geonetnz_quakes==0.5
+aio_geojson_geonetnz_quakes==0.9
 
 # homeassistant.components.ambient_station
 aioambient==0.3.1

